lib/factgrp.gi

```diff
@@ -1113,7 +1183,7 @@ local gimg,img,dom,b,improve,bp,bb,i,k,bestdeg,subo,op,bc,bestblock,bdom,
 	    fi;
 	  else
 	    k:=KernelOfMultiplicativeGeneralMapping(op);
-	    SetSize(Range(op),Index(gimg,k));
+	    SetSize(Range(op),IndexNC(gimg,k));
```

Comment:
This line of code (and the code surrounding it) is not tested by the Travis test suite.